code/src/utest/plugin/lsttokens/sizeadjustment/AbbTokenTest.java

```diff
@@ -75,15 +75,15 @@ public void testInvalidInputEmpty() throws PersistenceLayerException
 	public void testValidInputs() throws PersistenceLayerException
 	{
 		assertTrue(parse("Nieder�sterreich"));
-		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB));
+		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB_KR));

```

Comment:
Not sure if it matters, but there is a diamond question-mark symbol between "Nieder" and "sterreich"